brainframe_qt/ui/splash_screen/splash_screen.py

```diff
@@ -72,9 +81,14 @@ def show_configuration_button(self):
         self.setLayout(QVBoxLayout())
         button = QPushButton(self.tr("Configure"))
         button.setFocusPolicy(Qt.NoFocus)
-        # noinspection PyUnresolvedReferences
-        button.clicked.connect(
-            lambda: ServerConfigurationDialog.show_dialog(parent=self))
+        button.clicked.connect(self._open_server_config)
 
         self.layout().addWidget(button)
         self.layout().setAlignment(button, Qt.AlignBottom | Qt.AlignRight)
+
+    def _open_server_config(self) -> None:
+        ServerConfigurationDialog.show_dialog(parent=self)
+
+
+# Import has side-effects
+_ = qt_resources
```

Comment:
You know I'm quite a fan of this pattern, but I notice this is done twice. Is that necessary? I imagine that the side-effects are taken care of by whichever file gets evaluated by Python first.

Reply:
Yeah it can probably be moved into a centralized location so that it's only performed once

Reply:
https://github.com/aotuai/brainframe-qt/issues/146